GroupsFromLDAP.groovy

```diff
@@ -0,0 +1,71 @@
+import java.util.Hashtable
+import javax.naming.Context
+import javax.naming.NamingEnumeration
+import javax.naming.NamingException
+import javax.naming.directory.*
+import javax.naming.ldap.*
+import jenkins.model.Jenkins;
+import nectar.plugins.rbac.strategy.*;
+import hudson.security.*;
+import nectar.plugins.rbac.groups.*;
+import nectar.plugins.rbac.roles.*;
+
+
+try {
+        String ldapAdServer = "ldap://192.0.2.36:389"
+        String ldapSearchBase = "dc=example,dc=com"
+
+        String ldapUsername = "CN=tesla,CN=Users,DC=example,DC=com"
+        String ldapPassword = "Password12"
```

Comment:
For the same principle that the above comment, as a suggestion "ldapUsername" and "ldapPassword" could be changed for  "ldapManagerDN" and "password"